### PR TITLE
Displaying Cohort Reports

### PR DESCRIPTION
**Pull request by b7rch4**
Co-authored-by: richardtully richardtully12@gmail.com

Problem: We need to be able to display cohort reports

Solution: We have created a React component which fetches the required data from the backend, and then displays it on the page.
Cypress and React tests are also included.

Jira issue: https://dfacademy.atlassian.net/jira/software/projects/AT/boards/4?selectedIssue=AT-19

### CODE REVIEW

File: src/components/Reports/Reports.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        }
+    }
+
+    componentDidMount() {
+        fetch(`${process.env.REACT_APP_API_URL}/api/cohorts/${this.props.cohortId}/reports`)
+      .then(res => res.json())
+      .then(data => this.setState({reports: data.report}))
+    }
+
+    render() {
```
Comment (dearshrewdwit) [inline src/components/Reports/Reports.js:24]: This is a large component. 

Can you add an issue to your board that refactors this into smaller components according to report category maybe?

**Review by dearshrewdwit — APPROVED** [review]
Good. See comment.

File: src/components/Reports/Reports.js (hunk 1 above)
Comment (dearshrewdwit) [reply src/components/Reports/Reports.js:24]: You can decide whether you want to merge it now, and open a new PR with the refactor, or work on this PR some more.

Comment (FractalFraction) [reply src/components/Reports/Reports.js:24]: We will merge it now and then create a new PR for the refactor.

**Comment by FractalFraction** [comment]
Looks good to me!

**Review by FractalFraction — APPROVED** [review]
Looks good to me!

**Pull request merged**